introduction.md

```diff
@@ -1,2 +1,11 @@
 Hello my name is modeckimellett. That is all.
 My favorite Code School path is JavaScript
+Deadly Skills:
+* Java
+* Groovy
+* JavaScript
+* HTML
+* CSS
+* Git
+* Adobe CQ/Adobe AEM
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?